cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/helpers.js

```diff
@@ -0,0 +1,342 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Get specific row of DocTable.
+ * @param {number} rowNumber Integer starts from 0 for the first row
+ */
+export function getDocTableRow(rowNumber) {
+  return cy.getElementByTestId('docTable').get('tbody tr').eq(rowNumber);
+}
+
+/**
+ * Get specific field of DocTable.
+ * @param {number} columnNumber Integer starts from 0 for the first column
+ * @param {number} rowNumber Integer starts from 0 for the first row
+ */
+export function getDocTableField(columnNumber, rowNumber) {
+  return getDocTableRow(rowNumber).findElementByTestId('docTableField').eq(columnNumber);
+}
+
+/**
+ * find all Rows in Doc Table Field Expanded Document.
+ * @param expandedDocument cypress representation of the Doc Table Field Expanded Document
+ */
+export function findDocTableExpandedDocRowsIn(expandedDocument) {
```

Comment:
should we name it to getExpandedDocTableRows?

Reply:
I think findExpandedDocTableRows would be better to stick with the find naming convention used by cypress.